build/ci/macos/dumpsyms.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+echo "Generate dump symbols"
+
+GEN_SCRIPT=tools/crashdump/generate_syms.sh
+DUMPSYMS_BIN=$HOME/breakpad/dump_syms
+ARTIFACTS_DIR=build.artifacts
+BUILD_DIR=build.release
+SYMBOLS_DIR=$ARTIFACTS_DIR/symbols
+MSCORE_BIN=$BUILD_DIR/main/mscore.app/Contents/MacOS/mscore
+
+echo "GEN_SCRIPT: $GEN_SCRIPT"
+echo "DUMPSYMS_BIN: $DUMPSYMS_BIN"
+echo "BUILD_DIR: $BUILD_DIR"
+echo "SYMBOLS_DIR: $SYMBOLS_DIR"
+echo "MSCORE_BIN: $MSCORE_BIN"
+
+$GEN_SCRIPT --dumpsyms-bin $DUMPSYMS_BIN --build-dir $BUILD_DIR --symbols-dir $SYMBOLS_DIR --mscore-bin $MSCORE_BIN
+
+echo "-----"
+ls $SYMBOLS_DIR
```

Comment:
Seems this is where the CI errors out? If that is not intended an 'exit 0` after the 'ls` should help

Reply:
error with the wrong path to binary mscore
I added debug code, to find the error

Reply:
It still fails though at that `ls`, causing the script to end with a non-zero exit status